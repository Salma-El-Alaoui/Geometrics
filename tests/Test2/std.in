R r1 1 1 1 1
R r2 2 2 2 2
R r3 3 3 3 3
R r2 1 2 3 4
R r4 a 1 1 1
R r4 -1 -1 -1 -1
R r5 1 1
R r5 1 1 1 1 1 1
R 123 1 2 3 4
EXIT
