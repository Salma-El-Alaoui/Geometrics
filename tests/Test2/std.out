OK
#Element ajoute: r1
OK
#Element ajoute: r2
OK
#Element ajoute: r3
ERR
#Un element geometrique nomme r2 existe deja
ERR
#Parametres invalides
OK
#Element ajoute: r4
ERR
#Parametres invalides
ERR
#Parametres invalides
ERR
#Parametres invalides
